Shared/UI/Util/ButtonStyles.swift

```diff
@@ -16,6 +16,7 @@ public struct TableCellButtonStyle: ButtonStyle {
                 Color.selectedCell
                     .opacity(configuration.isPressed ? 1 : 0)
                     .padding(padding)
+                    .ignoresSafeArea()
```

Comment:
Can you make this:
```suggestion
                    .ignoresSafeArea(.container)
```

Recently have had a lot of issues with keyboards and `ignoreSafeArea`